src/style.css

```diff
@@ -0,0 +1,120 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  display: block;
+  margin: auto;
+  padding: 20px 8px;
+}
+
+.form.bigForm {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.bigImgBox {
+  display: flex;
+  align-items: center;
+}
+
+.bigSearchIcon {
+  position: absolute;
+  padding-left: 26px;
+  padding-bottom: 2px;
+  height: 19px;
+  width: 19px;
+}
+
+.bigSearchInput {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+}
+
+.bigSearchInput::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+/* input.smallSearchInput {
+  padding-top: 2px;
+} */
+
+.bigSearchInput:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigSearchInput:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir-Heavy, sans-serif;
+}
+
+.form.smallForm {
+  display: flex;
+}
+
+.smallImgBox {
+  display: flex;
+  align-items: center;
+}
+
+.smallSearchIcon {
+  position: absolute;
+  padding-left: 13px;
+  padding-bottom: 2px;
+  height: 11px;
+  width: 11px;
+}
+
+.smallSearchInput {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+}
+
+.smallSearchInput::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.smallSearchInput:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.smallSearchInput:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir-Heavy, sans-serif;
+}
```

Comment:
Omg, you duplicate a lot of styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd item